frontend/__tests__/src/features/boardSlice.test.tsx

```diff
@@ -0,0 +1,20 @@
+import boardReducer, { changeBoardVerifyState } from '@src/features/board/boardSlice'
+
+describe('board reducer', () => {
+  it('should false when handle initial state', () => {
+    const STEPPER = boardReducer(undefined, { type: 'unknown' })
```

Comment:
maybe the description update to 'should return false', and the constant update as stepper